apps/OboeTester/app/src/main/java/com/mobileer/oboetester/ReverseJniEngine.java

```diff
@@ -0,0 +1,95 @@
+package com.mobileer.oboetester;
+
+import android.util.Log;
+
+public class ReverseJniEngine {
+
+    private static final String TAG = "ReverseJniEngine";
+    private long mNativeEngineHandle = 0;
+    private double mPhase = 0.0;
+    private double mPhaseIncrement = 0.0;
+    private final int mSampleRate = 48000;
+    private final double mFrequency = 440.0; // A4 tone
+    private int mXRunCount = 0;
+    private int mSleepDurationUs = 0;
```

Comment:
Why sleep during data callback? It will cause underrun.

Reply:
This is done on purpose. This is to simulate an actual workload